apps/content/migrator/migrations/6/index.js

```diff
@@ -0,0 +1,60 @@
+const mongoose = require('mongoose')
```

Comment:
@davidmeirlevy I'm surprised to see plain js :P

Reply:
It was written more than 2 years ago, and it needs max flexibility (it's migrating DB structure).